.pip-tools.toml

```diff
@@ -0,0 +1,12 @@
+[tool.pip-tools]
+resolver = "backtracking"
+allow-unsafe = true
+strip-extras = true
+quiet = true
+output-file = "docs/docsite/requirements.txt"
+upgrade-package = [
+    "sphinx == 5.1.1",
+    "sphinx-ansible-theme == 0.9.1",
+    "docutils == 0.16",
+    "Jinja2 < 3.1",
```

Comment:
it seems like it would be much harder to get dependabot to upgrade packages in a list in a file like this. If not going the bot route, what would be the manual instructions for bumping the requirements?

Reply:
Call `pip-compile` under the same Python/arch/OS. Maybe, through a tox wrapper. I'm not sure if the dependabot have already upgraded the pip-tools version they use, by the way.

Reply:
Also, note that these are inputs so they aren't supposed to be updated manually at all. Only the lockfile should be generated. Everything else is human-maintained by design.

Reply:
I the other thread I also suggest dropping this from the config.